packages/editor-ui/src/components/HtmlEditor/HtmlEditor.vue

```diff
@@ -28,6 +28,7 @@ import { n8nCompletionSources } from '@/plugins/codemirror/completions/addComple
 import { editorKeymap } from '@/plugins/codemirror/keymap';
```

Comment:
Might be worth creating a new editor for CSS, I guess some of the features of the HTML editor are not relevant for CSS. I realize it's more work, so I'm also fine with this for now, we could consider refactoring later.